com.reprezen.swagedit.openapi3.tests/src/com/reprezen/swagedit/openapi3/assist/OpenApi3ContentAssistProcessorTest.xtend

```diff
@@ -139,16 +146,37 @@ class OpenApi3ContentAssistProcessorTest {
 
 		proposals = test.apply(processor, "2")
 		assertThat(proposals.map[(it as StyledCompletionProposal).replacementString], hasItems())
-		
+
 		proposals = test.apply(processor, "3")
 		assertThat(proposals.map[(it as StyledCompletionProposal).replacementString], hasItems())
-		
+
 		proposals = test.apply(processor, "4")
 		assertThat(proposals.map[(it as StyledCompletionProposal).replacementString], hasItems())
 
 		proposals = test.apply(processor, "5")
-		assertThat(proposals.map[(it as StyledCompletionProposal).replacementString],  
+		assertThat(
+			proposals.map[(it as StyledCompletionProposal).replacementString],
 			hasItems("int32", "int64", "float", "double", "byte", "binary", "date", "date-time", "password", "")
 		)
 	}
+
+// TODO
+//	@Test
+	def void testResponseStatusCode() {
+		val document = new OpenApi3Document(new OpenApi3Schema)
+		val test = setUpContentAssistTest('''
+			paths:
+			  /foo:
+			    get:
+			      responses:
+			        <1>
+		''', document)
+
+		val proposals = test.apply(processor, "1")
+		assertThat(
+			proposals.map[(it as StyledCompletionProposal).replacementString],
+			hasItems("200:")
```

Comment:
I suggested to add a `:` to the `replacementString and it's expected in the `replacementString` in this test!